### PR TITLE
On stream changes

### PR DESCRIPTION
**Pull request by HasturDev**
Everything we made on stream together

### CODE REVIEW

**Comment by ryanpeach** [comment]
WTF DID YOU DOOOO!

File: src/server/components.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from dataclasses import dataclass
+
+@dataclass
+class Inside_Planet_Location:
```
Comment (ryanpeach) [inline src/server/components.py:4]: Why

File: Information.txt [hunk 2]
```diff
@@ -0,0 +1,25 @@
+1. Characters start on surface of planet
```
Comment (HasturDev) [inline Information.txt:1]: This sucks

Comment (HasturDev) [inline Information.txt:1]: I agree with the first comment

**Comment by ryanpeach** [comment]
Approved

**Review by ryanpeach — APPROVED** [review]
Approved

**Review by ryanpeach — APPROVED** [review]

**Pull request merged**